### PR TITLE
feat: correct category buttons

### PR DESCRIPTION
**Pull request by matijakljajic**

### CODE REVIEW

File: src/lib/components/Header.svelte [hunk 1]
```diff
@@ -1,48 +1,18 @@
...
-					<img
-						class="col-start-1 row-start-1 h-16 w-16 mr-6 md:mr-8 pointer-events-none"
-						src={logo}
-						alt="logo"
-					/>
-				</a>
-			</div>
-			<Searchbox bind:query />
+			<Searchbox categories={true} bind:query />
 			<div class="w-0 md:w-16 md:ml-8" />
```
Comment (aleksasiriski) [inline src/lib/components/Header.svelte:15]: Move this to Searchbox inside if categories statement

File: src/lib/components/Searchbox.svelte [hunk 2]
```diff
@@ -42,26 +70,48 @@
...
+						stroke-linecap="round"
+						stroke-miterlimit="10"
+						stroke-width="32"
+						d="M338.29 338.29L448 448"
+					/>
+				</svg>
+			</button>
+		</div>
+	</div>
+	{#if categories == true}
```
Comment (aleksasiriski) [inline src/lib/components/Searchbox.svelte:97]: No need for ==true

File: src/lib/components/Searchbox.svelte [hunk 3]
```diff
@@ -1,30 +1,58 @@
...
-		autocapitalize="none"
-		spellcheck="false"
-		autocorrect="off"
-		dir="auto"
-		{autofocus}
-		bind:value={query}
-	/>
-	<!-- <button
+	<div class="flex w-full justify-center items-center">
+		{#if categories == true}
```
Comment (aleksasiriski) [inline src/lib/components/Searchbox.svelte:25]: No need for ==true

File: src/lib/functions/categoryFrom.ts [hunk 4]
```diff
@@ -1,12 +1,19 @@
 import { CategoryEnum } from '$lib/types/category';
 
 export function categoryFrom(query: string): string | undefined {
```
Comment (aleksasiriski) [inline src/lib/functions/categoryFrom.ts:3]: This never returns undefined now

File: src/lib/functions/categoryFrom.ts [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
-	for (const category in CategoryEnum) {
-		if (query.startsWith(`!${category}`)) {
-			return category;
+	let gottenCategory: string = 'general';
```
Comment (aleksasiriski) [inline src/lib/functions/categoryFrom.ts:4]: No need for : string since = '' already does that

Comment (aleksasiriski) [inline src/lib/functions/categoryFrom.ts:4]: Maybe name it "categoryFromQuery" or "cat" (the latter is used in backend when reading from query)

File: src/lib/components/Searchbox.svelte [hunk 6]
```diff
@@ -1,30 +1,58 @@
...
 	export let query: string;
 	export let autofocus: boolean = false;
+	export let categories: boolean = false;
+
+	// functions
+	import { categoryEquals, categoryFrom, hasCategory } from '$lib/functions/categoryFrom';
+
+	// types
+	import { CategoryEnum } from '$lib/types/category';
+	import { quadIn } from 'svelte/easing';
```
Comment (aleksasiriski) [inline src/lib/components/Searchbox.svelte:15]: Unused import

File: src/lib/components/Searchbox.svelte [hunk 7]
```diff
@@ -1,30 +1,58 @@
...
+	// assets
+	import logo from '$lib/assets/logo.svg';
+
 	// parameters
 	export let query: string;
 	export let autofocus: boolean = false;
+	export let categories: boolean = false;
+
+	// functions
+	import { categoryEquals, categoryFrom, hasCategory } from '$lib/functions/categoryFrom';
```
Comment (aleksasiriski) [inline src/lib/components/Searchbox.svelte:11]: categoryEquals unused import

**Review by aleksasiriski — CHANGES_REQUESTED** [review]
Great work! Please just fix the few little things mentioned and remove all unused imports.

**Comment by matijakljajic** [comment]
@aleksasiriski todo: Rename currently named _Image_ category to _Images_.

**Review by aleksasiriski — APPROVED** [review]

**Pull request merged**